volumes/short_term_counting_program/README.md

```diff
@@ -114,6 +115,10 @@ Internal to the Transportation Data & Analytics team, data flows from legacy Ora
 
 Look in the `traffic` schema for all ad-hoc data tables.
 
+## Where can I find what data?
+
+!['where-to-find-data'](../img/where-to-find-adhoc-count-data.png)
+
```

Comment:
Can add a note here that the `traffic_` tables have `traffic_` stripped from them when they are transferred to the bigdata `traffic` schema. These tables are:

arterydata
cnt_det
countinfo
countinfomics
det
arc_link
category
node

Reply:
@radumas I updated the `traffic_` to `traffic.` so I'm not sure this note is necessary anymore. I think it might introduce more confusion than clarify things?

Reply:
I was thinking more in terms of identifying the tables that don't have a prefix (`atr_` or `tmc_`) which are being actively updated, but maybe this leads into a broader discussion of cleaning up the `traffic` schema.